src/client/swagger/unity.yml

```diff
@@ -23,6 +23,35 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  '/me/{id}':
```

Comment:
Do we need an `id` here? Can we assume that this will be for the currently-authenticated user using the userID/orgID in the session?

Reply:
Nah, I cut it out after Gunnar's suggestion about using the cookie as an identifier